src/app/utils.js

```diff
@@ -100,3 +101,12 @@ export const getUnitStatusGroup = unit => {
   };
   return response;
 };
+
+/**
+  Returns owner string from ownerTag
+  @param {string} ownerTag The ownerTag identifier returns from the API
+  @returns {string} The simplified owner string
+*/
+export const extractOwnerName = tag => {
+  return tag.split("@")[0].replace("user-", "");
```

Comment:
I can't really see a developer calling this function without supplying what they think is a user tag. And even if they did it would fail early in testing.

The real case here is making sure that the data the application is using is reliable and we may be able to make some improvements there. But as it stands right now I'm not sure adding a null check here would be very useful.